presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushProjectionThroughExchange.java

```diff
@@ -35,6 +35,35 @@
 import java.util.Map;
 import java.util.Optional;
 
+/**
+ * Transforms:
+ *
+ *  <pre>
+ *  PROJ(x = e1, y = e2)
```

Comment:
use Project instead of PROJ.

Please do not use capitalized names. You do not need to shout. :)